explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -690,27 +719,29 @@ object TargetTabContents extends TwoPanels:
                       // Apply the obs to selected targets on the tree
                       optViewAgwo
                         .map(agwov =>
-                          applyObs(id.idSet.toList,
-                                   treeTargets,
-                                   loading,
-                                   agwov,
-                                   ctx,
-                                   props.listUndoStacks,
-                                   props.expandedIds
+                          applyObs(
```

Comment:
In `val treeTargets` above I think you want to get rid of the `getOrElse` because `selectedIds` is more accurate than `props.focused.target` isn't it? Also, I think if you replaced that `getOrElse` with `selectedIds` you could get rid of this `if/else`.

Reply:
Let's try that.